File: Intro_to_CS/Introduction-to-Computer-Science-and-Programming-using-Python/Week1/PSET1/problem1.py
"""
Problem 1
0.0/10.0 points (graded)
Assume s is a string of lower case characters.

Write a program that counts up the number of vowels contained in the string s. 
Valid vowels are: 'a', 'e', 'i', 'o', and 'u'. 
For example, if s = 'azcbobobegghakl', your program should print:

"""

vowels = ["a", "e", "i", "o", "u"]
s = 'azcbobobegghakl'
count = 0

for i in range(len(s)):
	if s[i] in vowels:
		count += 1

print(count)

File: Intro_to_CS/Introduction-to-Computer-Science-and-Programming-using-Python_JS_Edition/Week1/PSET1/problem1.js
/*
Problem 1
0.0/10.0 points (graded)
Assume s is a string of lower case characters.

Write a program that counts up the number of vowels contained in the string s. 
Valid vowels are: 'a', 'e', 'i', 'o', and 'u'. 
For example, if s = 'azcbobobegghakl', your program should print:

*/

var vowels = ['a','e','i','o','u'];
var s = 'azcbobobegghakl';
var count = 0;

for (var i = 0; i < s.length; i++) {
    for (var j = 0; j < vowels.length; j++) {
        if (s.charAt(i) == vowels[j]) {
			count++;
			break;
		  }
    }
}

console.log("Number of vowels: " + 5);
